viYarn.xcodeproj/xcuserdata/dmitry.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -90,7 +90,7 @@
             ignoreCount = "0"
             continueAfterRunningActions = "No"
             filePath = "viYarn/Episode/view/EpisodeViewController.swift"
-            timestampString = "559228921.340698"
+            timestampString = "559414288.6995521"
```

Comment:
p0: we should add default .gitignore for xcode projects.
See example online, on github (it creates by deafult for iOS projects) or take it from any other of our projects.

Reply:
basically, we shouldn't have xcuserdata in here. no need to review your break points :)

@Rieon  - also start doing self reviews.